### DISCUSSION TITLE
C Indenting

### DISCUSSION THREAD

**Discussion by matfk** [helix-editor/helix#7294] [General]
Im having trouble getting the right 4 space indenting in C.
I only get the 2 space indenting when I use the editor.
Could really appreciate an answer or explanation for this.
This is the code I have in the `language.toml` file.

``` 
[[language]]
name = "c"
auto-format = false
indent = { tab-width = 4, unit = " " }
```

**Comment by pascalkuthe** [comment]
Helix automatically detects the indentation of existing files so if you are editing a file that already uses double space it will use double space no matter what your settings are

**Reply by dogue** [reply]
Is there any way to override this?
